Jenkinsfile

```diff
@@ -65,6 +65,10 @@ node {
     archiveArtifacts 'build/reports/**/*.html'
     notifyBuildFailure channel: channel
     throw err
-  }
+  } finally {
+        step([$class: 'InfluxDbPublisher',
+               customProjectName: 'CMC Draftstore',
```

Comment:
I'm not sure where this value is used but draft-store is now a common component and it may be best to have a naming convention in influx which matches the repo name?

Reply:
yes, please remove the CMC from this

Reply:
Removed the CMC from the name, and i'll make sure to fill up the PR template from now on Jason.